### PR TITLE
Update data link in `intro-1d` example

### LINKED ISSUE #295: Broken data link in introductory example
The link on line

https://github.com/JuliaGaussianProcesses/AbstractGPs.jl/blob/95c52f5b42bbfd392e9186778404e3a8b5ef69fe/examples/intro-1d/script.jl#L19

is broken and should point to

https://github.com/GPflow/GPflow/blob/develop/doc/sphinx/notebooks/basics/data/regression_1D.csv

### PR DESCRIPTION
**Pull request by jvaverka**
Fixes #295

**Summary**
Fix 404

**Proposed changes**
* update link to data

**Breaking changes**
None

### CODE REVIEW

**Review by willtebbutt — APPROVED** [review]
Thanks for sorting this. Will merge when CI passes (not this should affect anything).

**Review by devmotion — COMMENTED** [review]
Can we link to a specific revision to avoid that it becomes broken again?

**Comment by jvaverka** [comment]
Excellent question that I wondered about, but didn't know the answer to 😅

File: examples/intro-1d/script.jl [hunk 1]
```diff
@@ -16,7 +16,7 @@ Random.seed!(42)  # setting the seed for reproducibility of this notebook
 #md nothing #hide
 
 # Load toy regression
-# [dataset](https://github.com/GPflow/GPflow/blob/develop/doc/source/notebooks/basics/data/regression_1D.csv)
+# [dataset](https://github.com/GPflow/GPflow/blob/develop/doc/sphinx/notebooks/basics/data/regression_1D.csv)
```
Comment (devmotion) [inline examples/intro-1d/script.jl:19]:
```suggestion
# [dataset](https://github.com/GPflow/GPflow/blob/7705cee6723f78066981f27954130daaede55dfc/doc/sphinx/notebooks/basics/data/regression_1D.csv)
```

**Review by devmotion — COMMENTED** [review]
You can get permanent links to specific revisions by clicking on "Copy permalink". I added a suggestion.

**Pull request merged**